app/scripts/lib/user-agent.js

```diff
@@ -31,6 +31,15 @@ define(function (require, exports, module) {
       },
 
       /**
+       * Check if on a desktop device
+       *
+       * @returns {Boolean}
+       */
+      isDesktop () {
+        return ! this.isAndroid() && ! this.isIos();
```

Comment:
I kinda sorta hoped someone would call me out on this. I want to keep a helper function, I just don't know what to call it.